lib/scrabble.rb

```diff
@@ -1,18 +1,35 @@
 class Scrabble
+  attr_reader :word
+
+  def initialize
+    @word = nil
+  end
 
   def score(word)
-    1
+    @word = word.upcase!
+    word = sub_points(@word)
+    word = add_points(word)
+  end
+
+  def sub_points(arg)
```

Comment:
Write this out to keep other people who read your code from having to think about what it does. At first when I saw `sub_points` I started thinking about points below something else, which of course doesn't make any sense, but it just happened to be the first thing that came to mind. `substitute_points` or even `score_letters` saves you from having to do any thinking about what the method does.